workspaces/tekton/plugins/tekton/src/components/PipelineRunLogs/PipelineRunLogViewer.tsx

```diff
@@ -74,3 +72,24 @@ export const PipelineRunLogViewer = ({ pod }: PipelineRunLogViewerProps) => {
     </>
   );
 };
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: true,
```

Comment:
Setting retry = true will infinitely retry failing requests. I think we should have a limit here.

Reply:
maybe `retry: 10`  would be a good limit?

Reply:
10 might be a long time, unsure how what the interval in between retries. maybe 5.